src/components/10-atoms/progress-bar/pw.ui.test.js

```diff
@@ -0,0 +1,55 @@
+const host = process.env.TEST_HOST_STORYBOOK_URL;
+const tag = 'axa-progress-bar';
+
+describe('Progress Bar', () => {
+  it('should render small', async () => {
+    await openProgressBar();
+
+    const fontSize = await page.$eval(
+      'p',
+      el => window.getComputedStyle(el).fontSize
+    );
+
+    expect(fontSize).toBe('16px');
+  });
+
+  it('should write text correctly', async () => {
+    await openProgressBar();
+
+    expect(
+      await page.textContent(
+        'article > div.a-progress-bar--label > axa-text > p'
+      )
+    ).toBe('hi I am a text');
+  });
+
+  it('should calculate the percantage correctly', async () => {
+    await page.goto(
+      `http://localhost:6006/iframe.html?id=components-progress-bar--progress-bar&knob-value=60&knob-max=300&knob-text=&knob-fullWidth=true&knob-small=true&viewMode=story`
+    );
+    await page.waitForSelector(tag);
+
+    const progressBar = await page.$(
+      'article > div.a-progress-bar--border > div'
+    );
+    expect(await progressBar.getAttribute('style')).toBe('width: 20%');
```

Comment:
Hm, maybe I have to take a look here. Setting this style just inline looks like a code-smell to me. Better us the `<style` tag.